### PR TITLE
Truck 5060

### PR DESCRIPTION
**Pull request by samuelmale**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by samuelmale** [comment]
TRUNK-5060
https://issues.openmrs.org/browse/TRUNK-5060

File: api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 	public void globalPropertyChanged(GlobalProperty newValue) {
 		allowedLocales = new LinkedHashSet<Locale>();
 		for (String allowedLocaleString : newValue.getPropertyValue().split(",")) {
-		//The try catch blocked has been removed..............
+		//The try catch blocked has been removed.
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java]: Is this change really about removing the try and catch blocks?

File: api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java [hunk 2]
```diff
@@ -33,7 +33,7 @@
 	public void globalPropertyChanged(GlobalProperty newValue) {
 		allowedLocales = new LinkedHashSet<Locale>();
 		for (String allowedLocaleString : newValue.getPropertyValue().split(",")) {
-		//The try catch blocked has been removed.
+		//The try catch blocked has been removed..............
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java]: Is this change really about removing the try and catch blocks?

File: api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java (hunk 1 above)
Comment (samuelmale) [inline api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java]: Then I think I didn't understand the ticket.
What was it about?

Comment (samuelmale) [inline api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java]: What was I expected to do?

Comment (samuelmale) [inline api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java]: or I was supposed to catch the NullPointerException ?

File: api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java [hunk 3]
```diff
@@ -33,17 +33,10 @@
 	public void globalPropertyChanged(GlobalProperty newValue) {
 		allowedLocales = new LinkedHashSet<Locale>();
 		for (String allowedLocaleString : newValue.getPropertyValue().split(",")) {
-			try {
+		//The try catch blocked has been removed.
```
Comment (wluyima) [inline api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java]: Can you get rid of this comment please? I find it redundant

File: api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java [hunk 4]
```diff
@@ -33,17 +33,10 @@
...
 				if (allowedLocale != null) {
-					allowedLocales.add(allowedLocale);
-				}
-			}
-			catch (Exception e) {
-				// bad locale spec? just ignore it. the UI should take care of
-				// guiding the user.
-			}
-		}
+					allowedLocales.add(allowedLocale);}
```
Comment (wluyima) [inline api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java]: Did you format this code? The closing bracket should have gone to the next line if you did

File: api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java [hunk 5]
```diff
@@ -33,17 +33,10 @@
 	public void globalPropertyChanged(GlobalProperty newValue) {
 		allowedLocales = new LinkedHashSet<Locale>();
 		for (String allowedLocaleString : newValue.getPropertyValue().split(",")) {
```
Comment (wluyima) [inline api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java]: Aren't you missing a closing tag?

**Comment by dkayiwa** [comment]
@samuelmale the ticket url on this pull request is not complete.

**Pull request closed without merging**